tusclient/uploader.py

```diff
@@ -102,8 +125,7 @@ def get_offset(self):
         resp = requests.head(self.url, headers=self.headers)
         offset = resp.headers.get('upload-offset')
         if offset is None:
-            msg = 'Attemp to retrieve offset fails with status {}'.format(resp.status_code)
-            raise TusCommunicationError(msg, resp.status_code, resp.content)
+            raise TusCommunicationError('', resp.status_code, resp.content)
```

Comment:
I leveraged on the default message that now exists [here](https://github.com/tus/tus-py-client/blob/retry-upload/tusclient/exceptions.py#L24). Of course we may choose to favor more specific messages. What do you think?